### ISSUE TITLE
Run Example in TPUs

### ISSUE THREAD

**Issue by charlielito** [poets-ai/elegy#29]
Create an example using TPUs for training. IT can be in Colab

**Comment by cgarciae** [comment]
@charlielito this is almost done right?

**Comment by charlielito** [comment]
> @charlielito this is almost done right?

Yeah. With the MNIST Example. But we were discussing maybe including a more complex example for TPU?
